Makefile

```diff
@@ -71,8 +71,8 @@ ci-functional-test: create-ci
 	(cp -r ./code/$(EXTENSION-LIBRARY-FOLDER-NAME)/build ./ci/functional-test)
 
 ci-upstream-integration-test:
-	(./code/gradlew -p code/upstream-integration-tests uninstallDebugAndroidTest)
-	(./code/gradlew -p code/upstream-integration-tests connectedDebugAndroidTest)
+	(./code/gradlew -p code/upstream-integration-tests uninstallDebugAndroidTest -PEDGE_ENVIRONMENT=value1 -PEDGE_LOCATION_HINT=value2)
+	(./code/gradlew -p code/upstream-integration-tests connectedDebugAndroidTest -PEDGE_ENVIRONMENT=value1 -PEDGE_LOCATION_HINT=value2)
```

Comment:
Since this is a todo for next task, can you revert the makefile changes from this pr